### PR TITLE
aws module update

### PR DESCRIPTION
**Pull request by mirii1994**
Please review my code @yyyogev

### CODE REVIEW

File: metricbeat-yml-script.py [hunk 1]
```diff
@@ -169,8 +169,8 @@ def _add_aws_shipping_data():
                 module_yaml[SINGLE_MODULE_INDEX]["metrics"] = []
                 for aws_namespace in aws_namespaces:
                     module_yaml[SINGLE_MODULE_INDEX]["metrics"].append({"namespace": aws_namespace})
-                    if aws_namespace.lower() == "aws/lambda":
-                        module_yaml[SINGLE_MODULE_INDEX]["metrics"][-1]["tags.resource_type_filter"] = "lambda"
+                    if aws_namespace.lower() == "aws/ec2":
+                        module_yaml[SINGLE_MODULE_INDEX]["metrics"][-1]["tags.resource_type_filter"] = "ec2:instance"
```
Comment (idohalevi) [inline metricbeat-yml-script.py]: @mirii1994 why not pulling tags for every service?

Comment (mirii1994) [reply metricbeat-yml-script.py]: fixed it now.

File: metricbeat-yml-script.py [hunk 2]
```diff
@@ -179,6 +179,21 @@ def _add_aws_shipping_data():
...
+    if service_name == "elb" or service_name == "applicationelb" or service_name == "networkelb":
+        tags_val = "elasticloadbalancing"
+    if service_name == "amazonmq":
+        tags_val = "mq"
+    if service_name == "efs":
+        tags_val = "elasticfilesystem"
+
+    return tags_val
+
+
```
Comment (idohalevi) [inline metricbeat-yml-script.py:196]: Is it possible to have a few tags from different services in this function? also, what will happen if tags_val = service_name?

Comment (mirii1994) [reply metricbeat-yml-script.py:196]: The function is being called during an iteration on the chosen services, so we only need one service for every call.
For most of the services, the tags_val is usually the AWS namespace in lowercase. Only in some services (those that are in "if" statements- elb, amazonmq, etc...) it's not like that. So the default is tags_val = service_name, unless it's one of the mentioned services.

Comment (idohalevi) [reply metricbeat-yml-script.py:196]: @mirii1994 What will happen if we use AWS namespace as tags_val and it's not a valid value for metricbeat? will metricbeat fail? 

If only one value is accepted I will use if/elif, drop tags_val and use return on each statement, and return service_name at the end of the function

Comment (mirii1994) [reply metricbeat-yml-script.py:196]: When the value of the `tags.resource_type_filter` is invalid it logs the following error, but continues working and sending metrics: `getResourcesTags failed, skipping region <region name>: error GetResources: InvalidParameterException: Unsupported service=<invalid_value> in ResourceTypeFilter - <invalid_value>`.

I'll change the function to be in if/elif structure you suggested.

Comment (idohalevi) [reply metricbeat-yml-script.py:196]: @mirii1994 Are we sure it keeps sending the metrics from the services it failed on?

Comment (mirii1994) [reply metricbeat-yml-script.py:195]: Yes. Also, made the structure change in the tags function :)

**Pull request merged**